src/pages/Detail/DetailContents/AboutPrice/AboutPrice.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import MyChart from './Chart/MyChart';
+import ModalPortal from '../../../../components/Modal/Portal/Portal';
+import Modal from '../../../../components/Modal/Modal';
+import { isLoginedState, setModal } from '../../../../recoil';
+import useSocket from '../../../../hooks/useSocket';
+import * as S from './AboutPrice.style';
+
+const AboutPrice = ({ info, productId }) => {
+  const isLogined = useRecoilValue(isLoginedState);
+  const [modalOpen, setModalOpen] = useRecoilState(setModal);
+  const [biddingPrice, setBiddingPrice] = useState(0);
+
+  const { currentPrice, myBiddingPrice, chartData } = useSocket(
+    0,
+    productId,
+    biddingPrice !== 0 && biddingPrice
```

Comment:
```suggestion
    biddingPrice !== 0 && biddingPrice
```

논리 곱 연산자(&&)는 좌항부터 판단해서 true면 우항을 평가하고 우항이 true면 우항의 값을 반환합니다.
반대로, 좌항의 값이 false이면 false 반환하고 종료합니다.

만약 biddingPrice가 0 이면 좌항에서 false로 평가되고 boolean `false`가 반환됩니다. 

useSocket에서 0보다 큰 조건으로 받고 있는데, number type끼리 비교가 아닌, false와 0이 비교됩니다.

biddingPrice가 0이면 0을 반환하고, 그게 아니면 biddingPrice를 반환하는 식으로 반환되는 type을 고려 해 보시는게 좋겠습니다

* 참고 : number type `0`은 암묵적으로 `false`로 평가되는 falsy한 값 입니다. 참고 해 보세요~